src/main/java/uk/gov/ons/ctp/response/action/export/scheduled/ExportScheduler.java

```diff
@@ -31,6 +32,7 @@ public ExportScheduler(
 
   /** Carry out scheduled actions according to configured cron expression */
   @Scheduled(cron = "#{appConfig.exportSchedule.cronExpression}")
+  @Transactional
```

Comment:
The transaction boundary should not be here - we specifically want to commit the transaction _before_ we release the lock, otherwise we will have exactly what we don't want, which is potentially two instances doing the same work because they can't see the other's transaction data until it's committed.

Reply:
`ExportProcessor` function `processExport` already has the `@Transactional` annotation, which is the right place to have the transaction boundary.

Reply:
fixed